templates/frontend/slot.html

```diff
@@ -85,7 +85,7 @@ <h2 class="mt-6 ml-1">Slot #{{ slot.slot_number }}</h2>
             </tr>
             <tr>
                 <th><i id="graffiti-tooltip" class="far fa-question-circle question-tooltip"></i>Graffiti</th>
-                <td>{{ slot.graffiti }}</td>
+                <td>{{ slot.graffiti }}{% with slot.graffiti|hex_to_string as graffiti_string %}{% if graffiti_string %} - {{ graffiti_string }}{% endif %}{% endwith %}</td>
```

Comment:
```suggestion
                <td>{% with slot.graffiti|hex_to_string as graffiti_string %}{% if graffiti_string %}{{ graffiti_string }}{% endif %}{% endwith %}</td>
```

Everything's working fine, but maybe we should consider excluding the hex string entirely.

Reply:
I think it's a safe option to remove it as there are no other use cases for the hex string other than messages atm.